tests/test_catalyst/test_configuration.py

```
@@ -13,9 +13,9 @@
 @pytest.fixture
 def mock_env_vars():
     original_environ = os.environ.copy()
-    RAGAAI_CATALYST_ACCESS_KEY = os.getenv("RAGAAI_CATALYST_ACCESS_KEY")
-    RAGAAI_CATALYST_SECRET_KEY = os.getenv("RAGAAI_CATALYST_SECRET_KEY")
-    RAGAAI_CATALYST_BASE_URL = os.getenv("RAGAAI_CATALYST_BASE_URL")
+    RAGAAI_CATALYST_ACCESS_KEY = os.getenv("CATALYST_ACCESS_KEY")
```

Ruff (0.8.2)

Local variable `RAGAAI_CATALYST_ACCESS_KEY` is assigned to but never used

Remove assignment to unused variable `RAGAAI_CATALYST_ACCESS_KEY`

(F841)